### PR TITLE
don't restrict species search to ncbi

### PR DESCRIPTION
**Pull request by n1mus**
- [ ] I updated the README.md docs to reflect this change.

For changes to the codebase:

- [x] I have written tests to cover this change.
- [x] This is not a breaking API change OR
- [ ] This is a breaking API change and I have incremented the API version and added a summary to CHANGELOG.md.

### CODE REVIEW

File: spec/test/stored_queries/test_fulltext_search.py [hunk 1]
```diff
@@ -1,5 +1,12 @@
 """
-Tests for the generic fulltext search
+Tests for stored queries involving a fulltext search:
+* Generic fulltext_search (should be used with caution because it can be slow and timeout at 60s)
+* Taxonomy taxonomy_search_species_strain
+* Taxonomy taxonomy_search_species_strain_no_sort
+
+The latter two are switched between depending on the length of the search text.
+These stored query tests  are all bundled in one test file because their original purpose is to do a species/strain
```
Comment (ialarmedalien) [inline spec/test/stored_queries/test_fulltext_search.py:8]: double space between `tests` and `are`

File: spec/stored_queries/taxonomy/taxonomy_search_species.yaml [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# DEPRECATED. See taxonomy_search_species_strains and taxonomy_search_species_strains_no_sort
```
Comment (ialarmedalien) [inline spec/stored_queries/taxonomy/taxonomy_search_species.yaml:1]: do you know if anyone / anything is actually using this?

**Review by ialarmedalien — APPROVED** [review]
LGTM

File: spec/stored_queries/taxonomy/taxonomy_search_species.yaml (hunk 2 above)
Comment (n1mus) [reply spec/stored_queries/taxonomy/taxonomy_search_species.yaml:1]: I really don't think so

**Pull request merged**